src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -136,6 +136,9 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._register(DOM.addDisposableListener(window, DOM.EventType.KEY_DOWN, (e: KeyboardEvent) => {
 			let event = new StandardKeyboardEvent(e);
 			if (this.isActive() && this.model.activeCell) {
+				if (e.repeat) {
```

Comment:
So now if you hold it down it'll scroll but not move through the cells? Because there were two ways that this could have been fixed : 

1. Holding down scrolls but doesn't move selection (what you have)
2. Holding down changes selected cell but doesn't move (except to bring new cell into view)

And it's not clear to me which one is better - but I was leaning towards the second since that seems useful so you don't have to click a whole bunch if you have a ton of cells.

Reply:
@chlafreniere @dzsquared Any thoughts here?

Reply:
When I tested holding down changes the selected cell by one and doesn't scroll.

Reply:
Just want to make sure that I'm parsing the difference here, but my expectation would be that holding the arrow keys does the same thing as tapping the arrow key many times.

i.e. if tapping the down arrow selects the next cell, I would expect that holding the down arrow for some period of time will select the next cell n times.

Reply:
Yeah ok I'm not sure this is the correct fix then. I'm not seeing where the event propagation is being stopped, that'd be where I'd start for this